server/monitor/src/main/resources/org/apache/accumulo/monitor/resources/js/navbar.js

```diff
@@ -21,7 +21,7 @@
 /**
  * The status options from the servers
  */
-const STATUS = {
+const SERVER_STATUS = {
```

Comment:
Can you avoid renaming these constants? It changes a lot more than needed, and makes it harder to review the changes.